examples/mount.rs

```diff
@@ -0,0 +1,22 @@
+#[macro_use] extern crate nickel;
+
+use nickel::{Nickel, Mount, StaticFilesHandler};
+
+fn main() {
+    let mut server = Nickel::new();
+
+    server.utilize(Mount::new("/a/b/c", middleware! {
+        "Hello world."
+    }));
+
+    server.utilize(Mount::new("/1/2/3", middleware! {
+        "H3110 w0r1d."
+    }));
+
+    server.utilize(Mount::new("/static/files/", StaticFilesHandler::new("examples/assets/")));
+    server.utilize(Mount::new("/static/files/a", middleware! {
+        "No static file called a!"
```

Comment:
Maybe a comment saying this is fall-through behavior, either way this is more useful than the top two usages :+1: (to me at least)